src/server/standalone.js

```diff
@@ -774,33 +772,19 @@ function StandAloneServer(gmeConfig) {
     logger.debug('creating basic static content related routing rules');
     //static contents
     //javascripts - core and transportation related files //TODO: remove config, middleware and bin
-    __app.get(/^\/(common|config|bin|middleware)\/.*\.js$/, function (req, res) {
-        expressFileSending(res, Path.join(__baseDir, req.path));
-    });
+    __app.get(/^\/(common|config|bin|middleware)\/.*\.js$/, Express.static(__baseDir, {index: false}));
 
     //TODO remove this part as this is only temporary!!!
-    __app.get('/docs/*', function (req, res) {
-        expressFileSending(res, Path.join(__baseDir, '..', req.path));
-    });
-
+    __app.get('/docs/*', Express.static(Path.join(__baseDir, '..'), {index: false}));
 
     __app.use('/rest/blob', BlobServer.createExpressBlob(middlewareOpts));
 
     //client contents - js/html/css
-    //stuff that considered not protected
-    __app.get(/^\/.*\.(css|ico|ttf|woff|js|cur)$/, function (req, res) {
-        expressFileSending(res, Path.join(__clientBaseDir, req.path));
-    });
+    __app.get(/^\/.*\.(css|ico|ttf|woff|js|cur)$/, Express.static(__clientBaseDir));
 
 
-    __app.get(/^\/.*\.(_js|html|gif|png|bmp|svg|json|map)$/, ensureAuthenticated, function (req, res) {
-        //package.json
-        if (req.path === '/package.json') {
-            expressFileSending(res, Path.join(__baseDir, '..', req.path));
-        } else {
-            expressFileSending(res, Path.join(__clientBaseDir, req.path));
-        }
-    });
+    __app.get('/package.json', ensureAuthenticated, Express.static(Path.join(__baseDir, '..')));
+    __app.get(/^\/.*\.(_js|html|gif|png|bmp|svg|json|map)$/, ensureAuthenticated, Express.static(__clientBaseDir));
```

Comment:
if you mean the static's extension parameter, then I think that would change the rule as it would try to load resources without extension as well.
Of course that probably not a big and bad change so if you feel, I can do it.